src/services/cleeng.checkout.service.ts

```diff
@@ -16,16 +18,13 @@ import { getOverrideIP } from '#src/utils/common';
 
 export const getOffers: GetOffers = async (payload, sandbox) => {
   const offers: Offer[] = [];
-  payload.offerIds.forEach(async (offerId) => {
-    const response = await getOffer({ offerId: offerId as string }, sandbox);
-
+  for (let i = 0; i < payload.offerIds.length; i++) {
```

Comment:
What was the reason for this change?

Reply:
because `Cleeng` has a separate endpoint for every offer( access fee in `InPlayer`), in the other hand, `InPlayer` has a single endpoint for all access fees( offers in `Cleeng`). and that is why we created a new method for fetching `Cleeng` offers in a single method( to simulate a single endpoint request )

Reply:
ahh if you mean for "`forEach` / `for`" change, this is because `forEach` is async and ignores the `await`. Maybe we can use `forEach`, but at the end, we have to wait for responses (e.g. `Promise.all(...)` )

Reply:
Oh gotcha. Indeed, let's try to use Promise.all so that they can be in parallel then.

Reply:
yes, I agree, it will be faster if they run in parallel

Reply:
updated.